apps/engineering/content/design/components/id.valueTruncate.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
```

Comment:
why is it a client component?

Reply:
Good point. Does not need to be. Changed here and on tooltip.onHover